apps/augury/src/app/error-page.tsx

```diff
@@ -0,0 +1,15 @@
+import { useRouteError } from 'react-router-dom';
+
+export default function ErrorPage() {
+  const error = useRouteError();
```

Comment:
I understand this is for demo, but let's remove it and create an actual 404 later.

Reply:
So it should just error out without a page for now?